indico/modules/events/abstracts/controllers/management.py

```diff
@@ -143,26 +120,28 @@ def _process(self):
         return jsonify_form(form=form, submit=_('Judge'), disabled_until_change=False)
 
 
-class RHAbstractNotificationLog(RHManageAbstractBase):
-    def _checkProtection(self):
-        RHManageAbstractBase._checkProtection(self)
-        if not self.abstract.can_judge(session.user, check_state=False):
-            raise Forbidden
+class RHAbstractNotificationLog(RHAbstractBase):
+    def _check_abstract_protection(self):
+        return self.abstract.can_judge(session.user)
 
     def _process(self):
         return WPManageAbstracts.render_template('abstract/notification_log.html', self._conf, abstract=self.abstract)
 
 
-class RHAbstractExportPDF(RHManageAbstractBase):
+class RHAbstractExportPDF(RHAbstractBase):
+    def _check_abstract_protection(self):
+        return self.abstract.can_see_reviews(session.user)
+
     def _process(self):
         pdf = ConfManagerAbstractToPDF(self.abstract)
         file_name = secure_filename('abstract-{}.pdf'.format(self.abstract.friendly_id), 'abstract.pdf')
         return send_file(file_name, pdf.generate(), 'application/pdf')
 
 
-class RHAbstracts(RHManageAbstractsBase):
-    """Display abstracts management page"""
+class RHAbstractsDashboard(RHManageAbstractsBase):
+    """Dashboard of the abstracts module"""
 
+    # Allow access even if the feature is disabled
```

Comment:
Just to make it clear why we set it to None. And it's so we can show a nice message how to enable the feature instead of hiding it from the menu or enabling it by default in all conferences (it was there before, I just added the comment)